### ISSUE TITLE
Failing to install with composer

### ISSUE THREAD

**Issue by slovenianGooner** [nunomaduro/phpinsights#74]
I seem to have a problem installing via composer.

> Problem 1
    - Installation request for nunomaduro/phpinsights ^1.0 -> satisfiable by nunomaduro/phpinsights[v1.0.0, v1.0.1, v1.0.2, v1.0.3, v1.0.4, v1.0.5].
    - Can only install one of: squizlabs/php_codesniffer[3.x-dev, 3.3.2].
    - Can only install one of: squizlabs/php_codesniffer[3.x-dev, 3.3.2].
    - Can only install one of: squizlabs/php_codesniffer[3.x-dev, 3.3.2].
    - Conclusion: install squizlabs/php_codesniffer 3.x-dev
    - Installation request for squizlabs/php_codesniffer 3.3.2 -> satisfiable by squizlabs/php_codesniffer[3.3.2].

> 

Gets stuck here.

**Comment by canvural** [comment]
I have similar issue

```
Problem 1
    - Installation request for slevomat/coding-standard (locked at 4.8.7) -> satisfiable by slevomat/coding-standard[4.8.7].
    - nunomaduro/phpinsights v1.0.0 requires object-calisthenics/phpcs-calisthenics-rules ^3.5 -> satisfiable by object-calisthenics/phpcs-calisthenics-rules[v3.5.0].
    - nunomaduro/phpinsights v1.0.1 requires object-calisthenics/phpcs-calisthenics-rules ^3.5 -> satisfiable by object-calisthenics/phpcs-calisthenics-rules[v3.5.0].
    - nunomaduro/phpinsights v1.0.2 requires object-calisthenics/phpcs-calisthenics-rules ^3.5 -> satisfiable by object-calisthenics/phpcs-calisthenics-rules[v3.5.0].
    - nunomaduro/phpinsights v1.0.3 requires object-calisthenics/phpcs-calisthenics-rules ^3.5 -> satisfiable by object-calisthenics/phpcs-calisthenics-rules[v3.5.0].
    - nunomaduro/phpinsights v1.0.4 requires object-calisthenics/phpcs-calisthenics-rules ^3.5 -> satisfiable by object-calisthenics/phpcs-calisthenics-rules[v3.5.0].
    - nunomaduro/phpinsights v1.0.5 requires object-calisthenics/phpcs-calisthenics-rules ^3.5 -> satisfiable by object-calisthenics/phpcs-calisthenics-rules[v3.5.0].
    - Conclusion: don't install object-calisthenics/phpcs-calisthenics-rules v3.5.0
    - Installation request for nunomaduro/phpinsights ^1.0 -> satisfiable by nunomaduro/phpinsights[v1.0.0, v1.0.1, v1.0.2, v1.0.3, v1.0.4, v1.0.5].
```

**Comment by ibrunotome** [comment]
@slovenianGooner this is my require-dev working normally

```json
  "require-dev": {
    "filp/whoops": "^2.0",
    "mockery/mockery": "^1.0",
    "nunomaduro/larastan": "^0.3.16",
    "nunomaduro/phpinsights": "^1.0",
    "object-calisthenics/phpcs-calisthenics-rules": "^3.4",
    "phpunit/phpunit": "^8.0",
    "sebastian/phpcpd": "^4.1",
    "squizlabs/php_codesniffer": "3.4.*"
  } 
```

**Comment by nunomaduro** [comment]
@slovenianGooner You still have this issue?

**Comment by slovenianGooner** [comment]
Will try today at work.

**Comment by ibrunotome** [comment]
@nunomaduro Which of the above dependencies are expendable after installation of phpinsights? To avoid double checks.

**Comment by nunomaduro** [comment]
@ibrunotome The ones that are already on  `https://github.com/nunomaduro/phpinsights/blob/master/composer.json`'s require key.

**Comment by anil3a** [comment]
I am having same problem. 

I added `"nunomaduro/phpinsights": "^1.0"` to my `require-dev` and it didn't work for me, instead I got big bunch of errors.

Here is a list of Errors i got in my GIST file.
https://gist.github.com/anil3a/524104a4ce9f82d4491350410d9442f7

**Comment by mikeerickson** [comment]
Do you use docker. If so, I think that is the best method for using insights as nothing needs to be installed and works across any PHP project.

**Comment by anil3a** [comment]
Unfortunately, I don't use docker.

**Comment by nunomaduro** [comment]
This issue has been fixed on the version `v1.5.0`.
